src/utils/helper.rs

```diff
@@ -42,6 +44,37 @@ pub mod Util {
     pub const SGINFO: &str = "/usr/bin/sginfo";
     pub const JBOD_EXPORTER: &str = "/usr/bin/prometheus-jbod-exporter";
 
+    /// Returns an enum with true or false if a directory is empty
+    ///
+    /// This function verify if the directory is empty.
+    ///
+    /// # Arguments
+    ///
+    /// * `path` - a string reference
+    pub fn is_folder_empty(path: &str) -> io::Result<bool> {
+        Ok(fs::read_dir(path)?.take(1).count() == 0)
```

Comment:
An other small NIT, avoid to think about taking N/counting just try to take one if none it's empty.
```suggestion
        Ok(fs::read_dir(path)?.next().is_none())
```

Reply:
OK, I have changed this part to use .is_nonde().